### PR TITLE
Clarify input behaviours on zip:create

### PR DESCRIPTION
**Pull request by josevalim**

### CODE REVIEW

File: lib/stdlib/doc/src/zip.xml [hunk 1]
```diff
@@ -368,8 +368,11 @@
           <seeerl marker="erl_tar"><c>erl_tar</c></seeerl>.</p>
         <p><c><anno>FileList</anno></c> is a list of files, with paths relative 
           to the current directory, which are stored with this path in the
-          archive. Files can also be specified with data in binaries
-          to create an archive directly from data.</p>
+          archive. File system operations are perform to read the file metadata
```
Comment (jchristgit) [inline lib/stdlib/doc/src/zip.xml:371]: shouldn't this be "File system operations are perform*ed*"?

Comment (josevalim) [reply lib/stdlib/doc/src/zip.xml:371]:
```suggestion
          archive. File system operations are performed to read the file metadata
```

**Comment by jhogberg** [comment]
Thanks for the PR! It looks great, please squash it so it can be merged. :)

**Comment by josevalim** [comment]
Done!

**Pull request merged**